Code/Editor/ViewportTitleDlg.cpp

```diff
@@ -537,21 +537,21 @@ void CViewportTitleDlg::AddAspectRatioMenus(QMenu* menu, std::function<void(int,
 
     for (size_t i = 0; i < customPresets.size(); ++i)
     {
-        if (customPresets[i].isEmpty())
+        if (customPresets[static_cast<int>(i)].isEmpty())
```

Comment:
And same here, since `i` is not used in those loops

Reply:
sure, replacing